src/Component/Card/PokemonCard.tsx

```diff
@@ -14,6 +14,8 @@ const PokemonCard: React.FC<CardProps> = ( props  : CardProps ) => {
         if (!favorited) {
             setFavorite(true)
             chooseCard(card)
+        } else {
```

Comment:
good use of else here! using the same event handler makes sense here

Reply:
nice destructuring on line 11 in this component!